upstairs/src/dummy_downstairs_tests.rs

```diff
@@ -511,7 +511,7 @@ pub(crate) mod protocol_test {
     /// downstairs responds with a read response for a certain job, then more
     /// work is sent.
     #[tokio::test]
-    async fn test_flow_control() {
+    async fn _test_flow_control() {
```

Comment:
What's the leading `_` do? Did you mean to disable this test like `_test_successful_live_repair` below?

Reply:
I did mean to disable this test initially, and confused myself later when I saw it still failing.  

I'm really using this PR as a way to trigger the CI to run (the only place I can get it to fail) 
as I debug the failures.

Reply:
Can I be helpful in debugging it? Happy to pitch in if I can!